keyboards/preonic/keymaps/default/keymap.c

```diff
@@ -274,7 +274,7 @@ void dip_switch_update_user(uint8_t index, bool active) {
             } else {
                 muse_mode = false;
             }
-    xs}
+    }
```

Comment:
This was fixed in https://github.com/qmk/qmk_firmware/pull/6748... but no merge conflict?

Reply:
lol. Probably because it's the same change?